run_action.py

```diff
@@ -194,7 +213,7 @@ def generate_single_comment(
             "path": file_path,
             "line": end_line_num,
             "side": "RIGHT",
-            "body": f"{single_comment_marker} **{markdown(name)}** {single_comment_marker}\n"
+            "body": f"{single_comment_marker} {diagnostic_name_visual(name)} {single_comment_marker}\n"
```

Comment:
`pylint` [complains](https://github.com/platisd/clang-tidy-pr-comments/actions/runs/15794221405/job/44561815338?pr=94#step:5:13) about this line being too long.
Can you please either refactor this a bit to abide by the rule or suppress it? I am OK with either.